tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -96,9 +96,13 @@ fn run_tests<S: BlackBoxFunctionSolver>(
     )?;
 
     let test_functions = context.get_all_test_functions_in_crate_matching(&crate_id, test_name);
+    let count_all = test_functions.len();
+    if count_all == 0 {
+        return Err(CliError::Generic(format!("[{}] Found 0 tests matching input", package.name)));
```

Comment:
Would be nice to be able to print the arguments received as input args.

@jfecher Do you know how I might be able to do that? 

If not, I think this is sufficient.

Reply:
Do you mean input args to `nargo test`? There is an args object with the type `TestCommand` that gets passed to the main `run` function.

Reply:
Thanks, I managed to figure it out eventually 👍 

I think this update prints much nicer!

Reply:
I can tell mostly from the code but would you be able to post a screenshot of some example output?

Reply:
Yes, see the comment outside this conversation